pkg/pkgmgr/rpm.go

```diff
@@ -81,8 +82,22 @@ func (st rpmDBType) String() string {
 
 // Depending on go-rpm-version lib for RPM version comparison rules.
 func isValidRPMVersion(v string) bool { // nolint:revive
-	// TODO: Verify if there are format correctness check that need to be added given lack of support in rpmVer lib
-	return true
+	err := isValidPackage(v)
+	return err == nil
```

Comment:
instead of isValidPackage maybe we could leave it in this function since we are just checking version?